all.json

```diff
@@ -14,6 +14,16 @@
 		"1inch-network.net",
 		"1inch-wallet.org",
 		"1inch.tk",
+		"bitgert.swap.defi-token.my.id",
+		"bridge.hertzswap.net",
```

Comment:
Nothing here ... yet. However `hertzswap.net` does seem like it connects, i.e. doesn't ask for seeds.

Reply:
Thanks for your vigilance, coffee has not kicked in yet 😴